.vscode/settings.json

```diff
@@ -12,7 +12,7 @@
     "debughandlers"
   ],
   // use ESLint to format code on save
-  "editor.formatOnSave": true,
+  "editor.formatOnSave": false,
```

Comment:
What's up with this?

Reply:
It was repeatedly breaking in my editor.

Reply:
It appears to happen in code spaces as well the on save and the formatting code command fight each other undoing each others changes